pkl-core/src/test/files/LanguageSnippetTests/input/api/bytes.pkl

```diff
@@ -0,0 +1,38 @@
+amends "../snippetTest.pkl"
+
+examples {
+  ["basic"] {
+    Bytes(List(1, 2, 3, 4))
+    Bytes("AQIDBA==")
+  }
+  ["equality"] {
+    Bytes(List()) == Bytes("")
+    Bytes(List(1, 2, 3, 4)) == Bytes("AQIDBA==")
+    Bytes(List(1, 2)) != Bytes(List(1, 2, 3, 4))
+  }
+  ["md5"] {
+    Bytes(List(1, 2, 3, 4)).md5
+    Bytes(List()).md5
+  }
+  ["sha256"] {
+    Bytes("AQIDBA==").sha256
+    Bytes("").sha256
+  }
```

Comment:
```suggestion
  }
  ["sha256Int"] {
    Bytes("AQIDBA==").sha256Int
    Bytes("").sha256Int
  }
```

We've added this to the `Bytes` class. We should add coverage for it.

Reply:
We'll probably want some tests for `encodeToStringWithCharset("UTF-8")` and `encodeToStringWithCharset("ISO-8859-1")`